arches_lingo/src/arches_lingo/components/generic/ComponentManager/ComponentManager.vue

```diff
@@ -0,0 +1,150 @@
+<script setup lang="ts">
```

Comment:
+1 for having something common like this. I think in a prior iteration we went back and forth on something like ConceptOrSchemeBase, but this is more fully fleshed out with actions that relate to specific places in the template 👍